integration-test/helpers/requests.js

```diff
@@ -29,17 +25,15 @@
  * @returns {Promise<function>}
  *   Function that clears logged requests (and in progress requests).
  */
-async function logPageRequests (page, requests, filter) {
+async function logPageRequests (page, requests, filter, isPlaywright = false) {
```

Comment:
Could we set `isPlaywright` inside the function by checking if `page instanceof Puppeteer.Page` or similar? Then we could avoid needing the optional parameter.

Reply:
I was trying to find a good way to do that, your suggestion work so I'm using that now.